modules/installer.nix

```diff
@@ -116,6 +121,7 @@ let cfg = config.nix-dabei; in
 
           ssh-keygen = "${config.programs.ssh.package}/bin/ssh-keygen";
           setsid = "${pkgs.util-linux}/bin/setsid";
+          ntpdate = "${pkgs.ntp}/bin/ntpdate";
```

Comment:
I am okay with ntpdate as it's well-known, just wondering if there were any benefits in relying on systemd-timesyncd - but that would probably require us to override (and therefore compile/cache) our own systemd.

Reply:
I didn't feel like pulling in too much of systemd. `ntpdate` is small and well-working utility. Chosen for reliability